pino.js

```diff
@@ -9,6 +9,7 @@ var noop = require('./noop')
 var pid = process.pid
 var hostname = os.hostname()
 var baseLog = flatstr('{"pid":' + pid + ',"hostname":"' + hostname + '",')
+var extend = require('util')._extend
```

Comment:
I think there is margin here to write a fast-all-natural object merger (just saying).